met-web/src/config.ts

```diff
@@ -64,20 +61,6 @@ const KC_CLIENT = getEnv('REACT_APP_KEYCLOAK_CLIENT');
 const KC_REALM = getEnv('REACT_APP_KEYCLOAK_REALM');
 const KC_ADMIN_ROLE = getEnv('REACT_APP_KEYCLOAK_ADMIN_ROLE');
 
-// App constants
-const ENGAGEMENT_PROJECT_TYPES: string[] = getEnv(
```

Comment:
lol yikes

Reply:
Yikes at the hard-coded nature of project types? :P

Reply:
Yes exactly